File: kimhyunji/PGS/Level 1/삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int[] n = new int[3]; //세 명의 학생번호
        
        for(int i=0; i<number.length; i++){
            n[0] = number[i]; //첫번째 학생
            for(int j=i+1; j<number.length; j++){
                n[1] = number[j]; //두번째 학생
                for(int k = j+1; k<number.length; k++){
                    n[2] = number[k]; //세번째 학생
                    if(n[0] + n[1] + n[2] == 0) answer++; //값이 0이면
                }
            }
        }
        return answer;
    }
}

File: baekyeonsu/삼총사.js
function solution(number) {
  let answer = 0;

  const length = number.length;
  for (let i = 0; i < length; i++) {
    for (let j = i + 1; j < length; j++) {
      for (let k = j + 1; k < length; k++) {
        if (number[i] + number[j] + number[k] === 0) answer++;
      }
    }
  }

  return answer;
}
